addons/l10n_co_pos/models/pos_order.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api
+
+
+class PosOrder(models.Model):
+    _inherit = 'pos.order'
+
+    @api.model
+    def create_from_ui(self, orders, draft=False):
```

Comment:
no need of modifying the return of create from UI, as an rpc will be done in the override of "post_push_order_resolve"